interactive-keymap.lisp

```diff
@@ -36,7 +36,10 @@
   (message "~S finished" name)
   (pop-top-map))
 
-(defmacro define-interactive-keymap (name (&key on-enter on-exit abort-if) &body key-bindings)
+(defmacro define-interactive-keymap (name (&key on-enter on-exit abort-if
+                                                (exit-on '((kbd "RET") (kbd "ESC")
```

Comment:
Minor style nit `(exit-on (list (kbd "RET" ...))` is preferable to `'((kbd "RET") ...)` as that way the default is the list of values instead of a the code that would produce the list of values.